rainforest/tests.go

```diff
@@ -45,8 +45,10 @@ type RFTest struct {
 	RFMLID      string                   `json:"rfml_id"`
 	Source      string                   `json:"source"`
 	Title       string                   `json:"title,omitempty"`
+	State       string                   `json:"state,omitempty"`
```

Comment:
To double-check: this is how it works in the API, right?

Reply:
@shosti Actually I think we need to change the API here. Sending a PUT request with the a state that the test already has returns a 400 (so if you upload a test twice without changing the state, 400). 👎

Reply:
This has been fixed 🎉